### PR TITLE
allow QuerySet.count() to query multiple collections

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: django_mongodb/features.py [hunk 1]
```diff
@@ -380,6 +379,8 @@ def django_test_expected_failures(self):
             "timezones.tests.NewDatabaseTests.test_query_datetimes_in_other_timezone",
         },
         "QuerySet.distinct() is not supported.": {
+            # TODO: add before merge
+            # "lookup.tests.LookupTests.test_lookup_collision_distinct",
```
Comment (WaVEV) [inline django_mongodb/features.py:383]: This todo is done?

**Comment by WaVEV** [comment]
I have been working on a similar task, specifically because the aggregate includes `count`. In my opinion, it is ok to merge this code, even though the majority of it will likely be moved or replaced.

**Review by WaVEV — CHANGES_REQUESTED** [review]
Minor thing: Solve the TODO

**Comment by timgraham** [comment]
The todo is because there's a corresponding patch to the Django fork that I'll merge in advance of this one:
```diff
diff --git a/tests/lookup/tests.py b/tests/lookup/tests.py
index a198a13b62..aa4ed46b98 100644
--- a/tests/lookup/tests.py
+++ b/tests/lookup/tests.py
@@ -1112,6 +1112,10 @@ class LookupTests(TestCase):
         self.assertEqual(Game.objects.filter(season__gt__in=[222, 333]).count(), 5)
         self.assertEqual(Game.objects.filter(season__gt__gt=111).count(), 5)
 
+    def test_lookup_collision_distinct(self):
+        # These were separated from the above test because distinct() isn't
+        # supported.
+
         # Players who played in 2009
         self.assertEqual(
   
```
I don't want to merge that until this is approved since it would cause other builds to fail in the meantime.

**Review by WaVEV — APPROVED** [review]

**Pull request merged**